src/Tickets/Site_Health/Subsections/Plugins/Plugin_Data_Subsection.php

```diff
@@ -404,7 +409,7 @@ private function get_number_of_attendees(): int {
 	 * @return array Associative array with formatted average, max, and min ticket prices.
 	 */
 	private function get_formatted_prices(): array {
-		$ticket_prices = tribe( 'tickets.event-repository' )->per_page( -1 )->where( 'has_tickets' )->pluck( 'cost' );
+		$ticket_prices = tribe( 'tickets.ticket-repository' )->per_page( -1 )->where( 'has_tickets' )->pluck( 'cost' );
```

Comment:
I think it should be event, as there are no `has_tickets` filter in the tickets repo.